cargo-dist/src/tasks.rs

```diff
@@ -1332,16 +1332,34 @@ impl<'pkg_graph> DistGraphBuilder<'pkg_graph> {
         let hint = format!("brew install {}", install_target);
         let desc = "Install prebuilt binaries via Homebrew".to_owned();
 
-        // If they have an x64 macos build but not an arm64 one, add a fallback entry
-        // to try to install x64 on arm64 and let rosetta2 deal with it.
+        // If they have an x64 macOS build but not an arm64 one, add a fallback entry
+        // to try to install x64 on arm64 and let Rosetta2 deal with it.
```

Comment:
If we're going to change the formatting of these, let's use the official spelling here and elsewhere in the file.
```suggestion
        // to try to install x64 on arm64 and let Rosetta 2 deal with it.
```